Source/WebCore/platform/ios/VideoFullscreenInterfaceAVKit.mm

```diff
@@ -557,12 +755,14 @@ - (BOOL)pictureInPictureWasStartedWhenEnteringBackground
 {
 #if PLATFORM(WATCHOS)
     return NO;
+#elif HAVE(PIP_CONTROLLER)
+    return [_pip pictureInPictureWasStartedWhenEnteringBackground];
 #else
     return _avPlayerViewController.get().pictureInPictureWasStartedWhenEnteringBackground;
```

Comment:
could make this consistent and use [_avPlayerViewController pictureInPictureWasStartedWhenEnteringBackground];

most of the the .get().blah got removed thorough webkit code already.

Reply:
Done. Updated a couple other places too.